src/sqlfluff/core/rules/std/L039.py

```diff
@@ -18,7 +18,7 @@ class Rule_L039(BaseCrawler):
 
     | **Best practice**
     | Unless an indent or preceeding a comment, whitespace should
-    be a single space.
+    | be a single space.
```

Comment:
This was actually complaining on linting, I imagine it wasn't caught before because of the issue